flink-python/src/main/java/org/apache/flink/streaming/api/runners/python/beam/BeamPythonStatelessFunctionRunner.java

```diff
@@ -170,34 +152,11 @@ public ExecutableStage createExecutableStage() throws Exception {
 			.build();
 	}
 
-	/**
-	 * Gets the proto representation of the input coder.
-	 */
-	private RunnerApi.Coder getInputCoderProto() {
-		return getRowCoderProto(inputType);
-	}
+	protected  abstract byte[] getUserDefinedFunctionsProtoBytes();
```

Comment:
The comment has not been addressed!
Keep one blank between protected and abstract.